usagov/usagov-2021
# Usagov 1722 make usagov logo responsive

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-

## Description

fixed casing error in image sourcing in top_nav.html.twig

## Type of Changes

- [x ] Bugfix
  - Add screenshot showing what it should look like

## Testing Instructions



run the site and make sure usagov logo acts the same as in prod, and double check lighthouse to make sure both "properly size images" and "explicite width and height" do not show up as failed audits

### Change Requirements



### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
